x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/watch/WatchStatusTests.java

```diff
@@ -28,27 +29,26 @@
 import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
-import static org.joda.time.DateTime.now;
 
 public class WatchStatusTests extends ESTestCase {
 
     public void testAckStatusIsResetOnUnmetCondition() {
         HashMap<String, ActionStatus> myMap = new HashMap<>();
-        ActionStatus actionStatus = new ActionStatus(now());
+        ActionStatus actionStatus = new ActionStatus(ZonedDateTime.now());
```

Comment:
I know these are only tests, but we can we avoid using the default TZ ? For anyone that looks at the unit tests for examples, this can lead to bugs.